Description: fixes a buffer overflow in svg processing.

---
Origin: upstream, http://fontforge.cvs.sourceforge.net/viewvc/fontforge/fontforge/fontforge/svg.c?r1=1.92&r2=1.93 
Bug-Debian: http://bugs.debian.org/550120
Last-Update: 2010-11-28

Index: fontforge-0.0.20100501/fontforge/svg.c
===================================================================
--- a/fontforge/svg.c
+++ b/fontforge/svg.c
@@ -150,7 +150,7 @@
 static int svg_pathdump(FILE *file, SplineSet *spl, int lineout,
 	int forceclosed, int do_clips) {
     BasePoint last;
-    char buffer[60];
+    char buffer[85];
     int closed=false;
     Spline *sp, *first;
     /* as I see it there is nothing to be gained by optimizing out the */
